docs/templates/feature.md

```diff
@@ -51,27 +50,77 @@ Potential *offline* types are:
 
 ### Online Storage
 
-The online storage is based on MySQL Cluster (NDB) and hence, any
-[MySQL Data Type](https://dev.mysql.com/doc/refman/8.0/en/data-types.html)
+The online storage is based on RonDB and hence, any
+[RonDB Data Type](https://dev.mysql.com/doc/refman/8.0/en/data-types.html)
 can be leveraged.
 
-!!! note "Type Inference"
-    When creating a [feature group](feature_group.md) from a Spark DataFrame, without
-    providing a schema manually, the feature store will infer the schema with feature
-    types from the DataFrame.
-
 Potential *online* types are:
 
 ```SQL
 "None", "INT(11)", "TINYINT(1)", "SMALLINT(5)", "MEDIUMINT(7)", "BIGINT(20)",
 "FLOAT", "DOUBLE", "DECIMAL", "DATE", "DATETIME", "TIMESTAMP", "TIME", "YEAR",
-"CHAR", "VARCHAR(25)", "VARCHAR(125)", "VARCHAR(225)", "VARCHAR(500)",
-"VARCHAR(1000)", "VARCHAR(2000)", "VARCHAR(5000)", "VARCHAR(10000)", "BINARY",
-"VARBINARY(100)", "VARBINARY(500)", "VARBINARY(1000)", "BLOB", "TEXT",
-"TINYBLOB", "TINYTEXT", "MEDIUMBLOB", "MEDIUMTEXT", "LONGBLOB", "LONGTEXT",
-"JSON"
+"CHAR", "VARCHAR(n)", "BINARY", "VARBINARY(n)", "BLOB", "TEXT", "TINYBLOB",
+"TINYTEXT", "MEDIUMBLOB", "MEDIUMTEXT", "LONGBLOB", "LONGTEXT", "JSON"
 ```
 
+#### Complex online types
+
+Additionally to the *online* types above, Hopsworks allows users to store complex types (e.g. *ARRAY<INT>*) on the online feature store.
+Hopsworks will take care of transparently serializing the complex features and storing them as *VARBINARY* on the online feature store. The serialization happens when calling the [save()](../api/feature_group_api/#save), [insert()](../api/feature_group_api/#insert) or [insert_stream()](../api/feature_group_api/#insert_stream) methods. The deserialization will be executed when calling the [get_serving_vector()](../api/training_dataset_api/#get_serving_vector) method to retrieve data from the online feature store.
```

Comment:
Hopsworks will take care of transparently serializing  ->
Hopsworks serializes the complex features transparently  and stores them as *VARBINARY* in the online feature store.